pkgs/applications/science/biology/poretools/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, pythonPackages, fetchFromGitHub }:
+
+pythonPackages.buildPythonPackage rec {
+  pname = "poretools";
+  version = "0.6.0";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "arq5x";
+    rev = "e426b1f09e86ac259a00c261c79df91510777407";
```

Comment:
The last tag was a couple dozen commits behind, this commit is stable and more up to date